ui/raidboss/data/04-sb/dungeon/fractal_continuum_hard.txt

```diff
@@ -0,0 +1,173 @@
+### FRACTAL CONTINUUM (HARD)
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 00:0839:.*is no longer sealed/ window 5000 jump 0
+
+#~~~~~~~~~~~#
+# MOTHERBIT #
+#~~~~~~~~~~~#
+
+# -ii 27A3 27A7 27A9
+
+0 "Start" sync / 00:0839:The reality augmentation bay will be sealed off/
+
+10.1 "Electrochemical Transfer" sync /:Motherbit:27A4:/ window 10,5
+15.7 "--sync--" sync /:Prototype Bit:27AB:/
+18.8 "Aetherochemical Laser" # sync /:Prototype Bit:27AA:/
+20.1 "Diffractive Laser" sync /:Prototype Bit:27A8:/
+20.7 "--sync--" sync /:Prototype Bit:27AB:/
+22.8 "Aetherochemical Laser" # sync /:Prototype Bit:27AA:/
+29.8 "Allagan Gravity" sync /:Motherbit:27A6:/
+
+37.9 "Electrochemical Transfer" sync /:Motherbit:27A4:/
+43.5 "--sync--" sync /:Prototype Bit:27AB:/
+54.2 "Citadel Buster" sync /:Motherbit:27A5:/
+
+77.9 "Electrochemical Transfer" sync /:Motherbit:27A4:/ jump 10.1
+86.6 "Aetherochemical Laser"
+87.9 "Diffractive Laser"
+90.6 "Aetherochemical Laser"
+97.6 "Allagan Gravity"
+105.7 "Electrochemical Transfer"
+
+#~~~~~~~~~~~~~~~~~~~~#
+# THE ULTIMA WARRIOR #
+#~~~~~~~~~~~~~~~~~~~~#
+
+# -ii 2798
+
+# Unfortunately, we have no choice but to use game log lines here.
```

Comment:
Do I remember correctly that the boss tethers to a statue in the room?

Reply:
You do. Unfortunately, the statues have no names associated with them, and they spawn before the encounter begins:
```log
03|2020-10-05T02:10:36.0900000-04:00|4004b325||0|46|0|0||108|6358|57250|57250|0|10000|0|0|-359.124|240.2462|-3.493345|1.099545||7366ede01b5cc2e5f728e3168bf19d0d
03|2020-10-05T02:10:36.0900000-04:00|4004b326||0|46|0|0||108|6358|57250|57250|0|10000|0|0|-357.3463|262.6122|-3.493345|2.199043||f3d5d6ffc34dac1a16e22d361b7a6b91
03|2020-10-05T02:10:36.0900000-04:00|4004b327||0|46|0|0||108|6358|57250|57250|0|10000|0|0|-336.6625|271.0951|-3.493345|-3.001998||a55a002baa9ab26a6e8162f615bde2c1
```

These lines occur the moment the hallway leading to the boss chamber unlocks. I suppose it's possible we could check for those and do some magic to inject tether syncs? But I'm not 100% sure they spawn in a consistent order.

Reply:
Ah yeah, and I don't know that we want to trust that the floating point values are always the same here either.  Sometimes positions are slightly off.  Ok, sounds like game log is really the only option.  Thanks for talking it through.